ROADMAP.md

```diff
@@ -4,18 +4,19 @@ This document defines a high level roadmap for Karmada development and upcoming
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep Karmada progressing and shipping exciting and valuable features.
 
-## 2023 H1
-- Cluster scheduling group
-- Multiple schedulers support
-- Application-level failover
-- Multi-cluster HPA (Horizontal Pod Autoscaling)
-- Karmada operator
-- ResourceInterpreter include third-party CRD resources
-- Karmada playground
+## 2024 H1
+- Lazy mode of PropagationPolicy
+- Cluster Problem Detector(CPD) - Part one: Cluster condition-based remedy system
+- Scheduler Enhancement - enable scheduler estimator supports resource quota
```

Comment:
Does the scheduler estimator plan to support estimation of resources similar to GPU?

Reply:
The estimator doesn't care about the resource type, no matter it is CPU, memory or GPU. 
This feature was released within v1.9 that estimator can take ResourceQuota into account during estimating.

Reply:
@chaunceyjiang Do you have any further comments?

Note that the roadmap is subject to change over time, we can propose a change at any time if needed.